app/code/Magento/Catalog/view/adminhtml/ui_component/category_form.xml

```diff
@@ -353,7 +353,6 @@
                     <additionalClasses>
                         <class name="admin__field-small">true</class>
                     </additionalClasses>
-                    <addBefore>$</addBefore>
                     <label translate="true">Layered Navigation Price Step</label>
```

Comment:
Can we fix in such a way that declaration is handled properly instead?

Reply:
@orlangur can you please suggest ..I have one other option to do this with modifiers but modifiers are not there for category form so should I add new modifier for category form to add this in the Layered Navigation Price Step field . 
In ui component I also tried to do with helper attribute but helper attribute is not allowed there.

Reply:
@orlangur do you propose to introduce a specific UI component for price range?

Reply:
@sivaschenko I propose to introduce whatever entity with a declaration instead `<addBefore>$</addBefore>` which is able to display proper currency symbol.